torch/csrc/autograd/functions/accumulate_grad.cpp

```diff
@@ -83,14 +84,11 @@ variable_list AccumulateGrad::apply_with_saved(
   at::Tensor grad_copy = variable.grad();
   saved.before(variable_copy);
   saved.before(grad_copy);
-  accumulateGrad(
-      variable_copy,
-      grad_copy,
-      grads[0],
-      0 /* num_expected_refs, 0 disables aliased reuse */,
-      [&saved, this](const at::Tensor& grad_update) {
-        saved.assign_mutable_grad(variable, grad_update);
-      });
+  variable_copy.mutable_grad() = grad_copy;
+  static auto op = c10::Dispatcher::singleton()
```

Comment:
nit: If this is intentionally static for lifetime reasons, comment. I often find stuff like this gets lost in future refactors and programs subtly change.

Reply:
Isn't this the standard pattern for using this dispatcher?  Or is there another way to do it?

Reply:
not sure, static seems safe here, but still worth a comment.

Reply:
I added a comment